packages/form/translations/fr.yaml

```diff
@@ -46,7 +46,7 @@ caluma:
       greaterThanOrEqualTo: "La valeur indiquée dans ce champ ne doit pas être inférieure à {gte} "
       lessThanOrEqualTo: "L'entrée dans ce champ ne doit pas être supérieure à {lte} "
       notAnInteger: "Veuillez entrer un nombre entier"
-      inclusion: "'{value}' n'est pas une valeur valide pour ce champ"
+      inclusion: '"{label}" n''est pas une valeur valide pour ce champ'
```

Comment:
Double `'` so that the `n'est` is displayed correctly?

Reply:
Yes, `'` is the escape character in translations: https://formatjs.io/docs/core-concepts/icu-syntax/#quoting--escaping